### DISCUSSION TITLE
How to copy and paste multiline in meow

### DISCUSSION THREAD

**Discussion by vohv** [meow-edit/meow#276] [Q&A] [answered]
In vim:
![Kapture 2022-07-06 at 14 25 31](https://user-images.githubusercontent.com/28553596/177483203-f73aa6e6-2787-4bd3-8742-077bd9f2754f.gif)

I can do this operation by kmacro. Is there any easy way?

**Comment by DogLooksGood** [comment, marked as answer]
Use the built-in `rectangle-mode`.

Enable with `C-x SPC SPC`, then select, then `meow-save`, then `meow-yank`.
